test/blocks/accordion/accordion.test.js

```diff
@@ -33,15 +33,15 @@ describe('Accordion', () => {
     // handleClick()
     const firstAccordionButton = document.body.querySelector('dt button');
     expect(firstAccordionButton.getAttribute('aria-expanded')).to.equal('false');
-    expect(firstAccordionButton.getAttribute('daa-ll')).to.equal(`open-1|${'What if my dough didn\'t rise?'.trim().slice(0, 30)}|accordion`);
+    expect(firstAccordionButton.getAttribute('daa-ll')).to.equal(`open-1|${'What if my dough did'.trim().slice(0, 30)}`);
```

Comment:
Do we need to keep the `.trim().slice`?  The string is already trimmed and only 20 chars long.

Reply:
hmmm, I just kept what was in there and updated the value. As a unit test, it doesn't need either IMO.  Updated to simple string.